ckcp/openshift_dev_setup.sh

```diff
@@ -227,7 +227,12 @@ patches:
   # Copy the kubeconfig of kcp from inside the pod onto the local filesystem
   podname="$(kubectl get pods --ignore-not-found -n "$ns" -l=app=kcp-in-a-pod -o jsonpath='{.items[0].metadata.name}')"
   mkdir -p "$(dirname "$KUBECONFIG_KCP")"
-  kubectl cp "$APP/$podname:etc/kcp/config/admin.kubeconfig" "$KUBECONFIG_KCP" >/dev/null
+  # Wait until admin.kubeconfig file is generated inside ckcp pod
+  while [[ $(kubectl exec -n $APP $podname -- ls /etc/kcp/config/admin.kubeconfig >/dev/null 2>&1; echo $?) -ne 0 ]];do
+    echo -n "."
+    sleep 5
+  done
+  kubectl exec -n $APP $podname -- cat /etc/kcp/config/admin.kubeconfig > "$KUBECONFIG_KCP" 
```

Comment:
Move the `echo "OK"` line 220 after this line.
Also I think that there's a `kubectl cp` command.

Reply:
>Move the echo "OK" line 220 after this line.

Fixed

Reply:
@xinredhat I did not have an issue using kubectl cp with the same path /etc/kcp/config/admin.kubeconfig. Not sure where the /data folder is coming into picture for copying the kubeconfig.

```
++ kubectl exec -n ckcp ckcp-668bf6b867-xc28q -- ls /etc/kcp/config/admin.kubeconfig
++ echo 0
+ [[ 0 -ne 0 ]]
+ kubectl cp ckcp/ckcp-668bf6b867-xc28q:/etc/kcp/config/admin.kubeconfig /home/bnr/workspace/pipelines-service/ckcp/work/kubeconfig/admin.kubeconfig
+ echo OK

```

Reply:
`kubectl cp` modification is done